storage/perfschema/ha_perfschema.h

```diff
@@ -86,25 +86,25 @@ class ha_perfschema final : public handler
     Operations supported by indexes.
     None, there are no indexes.
   */
-  ulong index_flags(uint , uint , bool ) const
+  ulong index_flags(uint , uint , bool ) const override
   { return 0; }
 
-  uint max_supported_record_length(void) const
+  uint max_supported_record_length(void) const override
```

Comment:
In C++, the empty parameter list should be written as `()` and not `(void)`. This would be easy to check with the following:
```sh
git show|grep '^+.*(void)'
```